pages/contribution-flow.js

```diff
@@ -142,7 +142,7 @@ class NewContributionFlowPage extends React.Component {
       }
       return <ContributionBlocker blocker={contributionBlocker} account={account} />;
     } else if (step === 'success') {
-      return <ContributionFlowSuccess collective={account} />;
+      return <ContributionFlowSuccess collective={account} isCrypto={this.props.verb === 'crypto'} />;
```

Comment:
I don't remember `crypto` being a `verb`.

Reply:
Sorry, I missed this. I've corrected this now. 👍🏽